### PR TITLE
fix(jest): clean up tmp after exit

### PR DESCRIPTION
**Pull request by kevinhwang91**

### CODE REVIEW

File: jest.js [hunk 1]
```diff
@@ -2,14 +2,18 @@ const path = require('path')
...
 const fs = require('fs')
 
-process.on('uncaughtException', function (err) {
+process.on('uncaughtException', err => {
   let msg = 'Uncaught exception: ' + err.stack
   console.error(msg)
 })
 
+process.on('exit', () => {
+  fs.rmdirSync(process.env.TMPDIR, { recursive: true, force: true })
```
Comment (fannheyward) [inline jest.js:11]: This removes the whole TMPDIR, we should only remove `coc-test` that we created.

**Review by fannheyward — CHANGES_REQUESTED** [review]

File: jest.js (hunk 1 above)
Comment (kevinhwang91) [reply jest.js:11]: But on the line 21 have changed the ENV

**Review by fannheyward — APPROVED** [review]

**Comment by fannheyward** [comment]
Tested in my env, all tests passed but `error Command failed with exit code 1`.

**Comment by kevinhwang91** [comment]
Like your nodejs version is below 14. I must AFK now.

**Comment by fannheyward** [comment]
I'm using node 18.2.0

**Pull request merged**